Development/UID2SDKDevelopmentApp/UID2SDKDevelopmentApp/RootViewModel.swift

```diff
@@ -34,18 +32,15 @@ class RootViewModel: ObservableObject {
             UID2Settings.shared.environment = .custom(url: URL(string: "https://operator-integ.uidapi.com")!)
         }
 
+        return UID2Manager.shared
+    }()
```

Comment:
Is this a common pattern in Swift? Seems 'interesting' to me 😅. Maybe this could be a lazy instead?